src/app/(nextjs_migration)/(guest)/user/unsubscribe-monthly/page.tsx

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { redirect } from "next/navigation";
+import { updateMonthlyEmailOptout } from "../../../../../db/tables/subscribers";
+import { getL10n } from "../../../../functions/server/l10n";
+
+export default async function PublicScan(props: {
+  searchParams: { [key: string]: string | string[] | undefined };
+}) {
+  const token = props.searchParams.token;
+  if (typeof token !== "string") {
+    return redirect("/");
+  }
+  try {
+    await updateMonthlyEmailOptout(token);
+  } catch (error) {
+    console.log(error);
```

Comment:
Question (non-blocking): Will we need the log for tracing or should we remove it?

Reply:
The log was copied over from here: https://github.com/mozilla/blurts-server/blob/d4cb77862d91a61e0debf1032301428c2019ee04/src/controllers/unsubscribe.js#L26

So :shrug: I'd err towards keeping existing behaviour.